### DISCUSSION TITLE
Reset RX buffer on startReceive

### DISCUSSION THREAD

**Discussion by StevenCellist** [jgromes/RadioLib#853] [General]
Is there a possibility to reset the packetLength / rxBuffer? Ideally, these are reset whenever startReceive (or receive) is called, such that you get the info for the newest receive. Currently, if nothing is received, the old buffer remains intact and you cannot really tell that there was nothing new.

**Comment by StevenCellist** [comment]
Solved this in a better way and it appears to be impossible. Closing.

**Reply by jgromes** [reply]
Would you mind sharing how you resolved this in case anyone ever needs that information? :)

**Reply by StevenCellist** [reply]
Instead of starting a receive without timeout (either RxSingle or RxContinuous), I now start an RxSingle with a timeout. By including RxTimeout in the IRQ mask, you can retrieve whether there was a timeout or not after the timeout _should_ have fired a RxTimeout if RxSingle did not catch a message.
